Casks/r/rivet.rb

```diff
@@ -0,0 +1,28 @@
+cask "rivet" do
+  version "1.9.0"
+  sha256 "d5805b49f411a26fe8fda30fc855996a895f1169920dd4db551b5853130b16d8"
+
+  url "https://github.com/Ironclad/rivet/releases/download/app-v#{version}/Rivet.dmg", verified: "github.com/Ironclad/rivet/"
+  name "Rivet"
+  desc "Open-Source Visual AI Programming Environment"
+  homepage "https://rivet.ironcladapp.com/"
+
+  livecheck do
+    url "https://github.com/Ironclad/rivet/releases"
```

Comment:
```suggestion
    url :url
```

The `GithubReleases` strategy can generate the correct API URL from the cask `url`, so there's no need for a literal URL here.